### PR TITLE
MOSIP-10583 Upgrade Manual Adjudication

### PR DESCRIPTION
**Pull request by Shilpa-Manjunath**

### CODE REVIEW

File: registration-processor/core-processor/registration-processor-bio-dedupe-stage/src/main/java/io/mosip/registration/processor/biodedupe/stage/BioDedupeProcessor.java [hunk 1]
```diff
@@ -116,6 +116,9 @@
 
 	@Autowired
 	private Environment env;
+	
+	@Autowired
+	private BioDedupeStage biodedupestage;
```
Comment (MonobikashDas) [inline registration-processor/core-processor/registration-processor-bio-dedupe-stage/src/main/java/io/mosip/registration/processor/biodedupe/stage/BioDedupeProcessor.java:121]: from BioDedupeStage we would call BioDedupeProcessor and from here you are again calling stage.
Will it not create circular dependency? did u test it.

File: registration-processor/core-processor/registration-processor-manual-verification-stage/pom.xml [hunk 2]
```diff
@@ -125,6 +125,10 @@
 			<artifactId>micrometer-registry-prometheus</artifactId>
 			<version>${io.micrometer.prometheus.version}</version>
 		</dependency>
+		<dependency>
```
Comment (MonobikashDas) [inline registration-processor/core-processor/registration-processor-manual-verification-stage/pom.xml]: i think jackson dependency is already added in core and should be available in this stage, kindly cross check

File: registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/ManualVerificationApplication.java [hunk 3]
```diff
@@ -28,8 +28,17 @@ public static void main(String[] args) {
...
 		configApplicationContext.refresh();
+		try {
+		configApplicationContext.getBean(Listener.class).runQueue();
+		}catch (Exception e) {
+			e.printStackTrace();
+		}
 		ManualVerificationStage manualVerificationStage = configApplicationContext
 				.getBean(ManualVerificationStage.class);
 		manualVerificationStage.deployStage();
+		/*MessageDTO object = new MessageDTO();
```
Comment (MonobikashDas) [inline registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/ManualVerificationApplication.java]: pls remove this code

File: registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/ManualVerificationApplication.java [hunk 4]
```diff
@@ -28,8 +28,17 @@ public static void main(String[] args) {
 				"io.mosip.registration.processor.core.kernel.beans",
 				"io.mosip.registration.processor.packet.storage.config");
 		configApplicationContext.refresh();
+		try {
+		configApplicationContext.getBean(Listener.class).runQueue();
+		}catch (Exception e) {
+			e.printStackTrace();
```
Comment (MonobikashDas) [inline registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/ManualVerificationApplication.java:38]: pls add proper logger as per reg-proc coding standard everywhere,

File: registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/response/dto/Analytics.java [hunk 5]
```diff
@@ -0,0 +1,11 @@
+package io.mosip.registration.processor.manual.verification.response.dto;
+
+import lombok.Data;
+
+@Data
+public class Analytics {
+	
+	private String key1 = "value1";
```
Comment (MonobikashDas) [inline registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/response/dto/Analytics.java:8]: there could be multiple key values which could be more than 2. can we make it dynamic

File: registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/response/dto/AnalyticsDTO.java [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+	private String primaryOperatorID;
+	
+	private String primaryOperatorComments;
+	
+	private String secondaryOperatorID;
+	
+	private String secondaryOperatorComments;
+	
+	private String key1="value1";
```
Comment (MonobikashDas) [inline registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/response/dto/AnalyticsDTO.java]: could be multiple key/values, dont restrict to 2

File: registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/resources/bootstrap.properties [hunk 7]
```diff
@@ -14,4 +15,16 @@ server.servlet.path=/registrationprocessor/v1/manualverification
...
-eventbus.port=5720
\ No newline at end of file
+eventbus.port=5720
+
+spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.show-sql=true
+
+activemq.message.format=text
+
+registration.processor.manual.adjudication.policy.id=mpolicy-default-adjudication
```
Comment (MonobikashDas) [inline registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/resources/bootstrap.properties:25]: remove from here, should be read from config server

File: registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/code/ApiName.java [hunk 8]
```diff
@@ -155,6 +155,9 @@
 
 	/** The idschemaversion. */
 	IDSCHEMAVERSION,
+  
+  /** The cnienumber. */	
+	CNIENUMBER,
```
Comment (MonobikashDas) [inline registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/code/ApiName.java]: remove this

Comment (Shilpa-Manjunath) [reply registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/code/ApiName.java]: done

File: registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/ManualVerificationApplication.java (hunk 3 above)
Comment (Shilpa-Manjunath) [reply registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/ManualVerificationApplication.java]: Removed commented code

File: registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/ManualVerificationApplication.java (hunk 4 above)
Comment (Shilpa-Manjunath) [reply registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/ManualVerificationApplication.java:38]: Added logger

File: registration-processor/core-processor/registration-processor-manual-verification-stage/pom.xml (hunk 2 above)
Comment (Shilpa-Manjunath) [reply registration-processor/core-processor/registration-processor-manual-verification-stage/pom.xml]: Removed Dependency

File: registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/resources/bootstrap.properties (hunk 7 above)
Comment (Shilpa-Manjunath) [reply registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/resources/bootstrap.properties:25]: Placed in registration-processor-mz.properties file

**Pull request merged**